app/src/main/java/com/architectcoders/aacboard/model/ArasaacService.kt

```diff
@@ -0,0 +1,12 @@
+package com.architectcoders.aacboard.model
+
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface ArasaacService {
+
+    @GET("pictograms/es/search/{searchString}")
```

Comment:
provide locale as a path param?  @GET("pictograms/{locale}/search/{searchString}")

Reply:
I had locale as a path parameter but decided to remove it to simplify the API. I will include it again. I create a new issue to get locale value from the current device language